doc/user/download_backup.md

```diff
@@ -38,11 +38,11 @@ If backup storage type is "S3", users have to get the backup directly from S3.
 First of all, setup aws cli: https://aws.amazon.com/cli/ .
 
 Given the S3 bucket name that you passed to when starting etcd operator and the cluster name,
-backups are saved under prefix `${BUCKET_NAME}/v1/${NAMESPACE}/${CLUSTER_NAME}/` .
+backups are saved under prefix `${BUCKET_NAME}/${PREFIX}/v1/${NAMESPACE}/${CLUSTER_NAME}/` .
 
 List all backup files:
```

Comment:
Maybe change this to:


If no prefix is specified, list all backup files:

```
$ aws s3 ls <bucket-name>/v1/<namespace>/<cluster-name>/
2017-01-24 02:13:30    24608    3.1.0_0000000000000002_etcd.backup
```

If prefix is specified (add a link to the docs?), list backup files:
```
$ aws s3 ls <bucket-name>/<prefix>/v1/<namespace>/<cluster-name>/
```